appinfo/routes.php

```diff
@@ -22,11 +22,11 @@
  */
 
 return [
-	'routes' => [
+	'ocs' => [
 		[
 			'name' => 'video#getTracks',
-			'url' => '/video/tracks',
-			'verb' => 'GET'
+			'url' => '/video/v1.0/tracks',
```

Comment:
The JavaScript and PHP for the viewer plugin are updated together, so I do not immediately see the need for adding a version to this route.

Reply:
OK. I will remove the version.